config/jobs/kubernetes/client-go/client-go-presubmits.yaml

```diff
@@ -6,7 +6,7 @@ presubmits:
     path_alias: k8s.io/client-go
     spec:
       containers:
-      - image: golang:1.14.4
+      - image: golang:1.16
```

Comment:
I'd probably leave this at go1.15 for now, since that is what k8s.io/kubernetes is still on

Reply:
fixing.